firmware/arduino/chessboard/uart.ino

```diff
@@ -70,7 +95,14 @@ void serialEvent2()
                 // Sends acknowledgement
                 sendMessageToPi(currentState, buffer[5], errorCode);
 
+                movementFlag = true;
                 makeMove(buffer);
+
+                // Tell Pi that the chess timer button was pressed
+                if (transmitFlag)
+                    sendMessageToPi(END_TURN, 0, buffer[5]);
+                movementFlag = false;
```

Comment:
Can you please put this flag immediately after `makeMove` just for clarity's sake, ty